skills/declarative/calendarSkill/calendarskill.dialog

```diff
@@ -180,24 +180,24 @@
       "intent": "GetEventTitle",
       "actions": [
         {
-          "$kind": "Microsoft.SetProperties",
+          "$kind": "Microsoft.BeginDialog",
           "$designer": {
-            "id": "p5hdqN"
+            "id": "9n2omv"
           },
-          "assignments": [
-            {
-              "property": "dialog.query",
-              "value": "=turn.activity.text"
-            }
-          ]
+          "activityProcessed": true,
+          "dialog": "AuthenticationDialog"
         },
         {
           "$kind": "Microsoft.BeginDialog",
           "$designer": {
-            "id": "9n2omv"
+            "id": "ESk1DO"
           },
           "activityProcessed": true,
-          "dialog": "AuthenticationDialog"
+          "dialog": "ResolveDateTimeDialog",
+          "options": {
+            "query": "=turn.activity.text"
```

Comment:
The setProperty for the query property actually has to stay for scenarios when the user isn't authenticated yet when they call this dialog (through a VA for instance). If the authentication dialog requires a magic code, the turn.activity.text won't have the user's original utterance anymore

Reply:
OK, fixed